src/provisioner/provisioner.py

```diff
@@ -195,8 +200,68 @@ def cluster_bootstrap(cortx_conf_url: str = None):
                 cmd_proc = SimpleProcess(cmd)
                 _, err, rc = cmd_proc.run()
                 if rc != 0 or err.decode('utf-8') != '':
+                    CortxProvisioner._update_deployment_status(
+                        cortx_config_store, node_id, phase='deployment',
+                        status='error')
                     raise CortxProvisionerError(
                         rc, "%s phase of %s, failed. %s", interface,
                         components[comp_idx]['name'], err)
 
+                CortxProvisioner._update_deployment_status(
+                    cortx_config_store, node_id, phase='deployment',
+                    status='progress')
+
+        # Once the deployment is successful create VERSION.INFO
+        # file as a backup of RELEASE.INFO file.
+        CortxProvisioner._load_file(const.VERSION_FILE, const.VERSION_INDEX)
+        CortxProvisioner._copy_release_info()
+        CortxProvisioner._update_deployment_status(
+            cortx_config_store, node_id, phase='deployment', status='success',
+            version=const.VERSION_FILE)
+
         Log.info(f'Finished cluster bootstrap on {node_id}:{node_name}')
+
+
+    @staticmethod
+    def _load_file(file_name, index):
```

Comment:
its file_path and not just the file_name.

Reply:
Instead extending functions like these, we should extend classes. 
Can you add a new class name

```
class Manifest: (This class may be defined in the utils class already)
    pass

class CortxReleaseInfo(Manifest):
    def __init__():
        load release info

    def backup():
        create backup copy

....
```

Reply:
Done.